plaid/resources/plaid.toml

```diff
@@ -117,6 +117,25 @@ allowed_rules = ["testing_test.wasm"]
 client_id = 99999
 secret_key = ""
 
+# KMS USING IAM
+
+# [apis."aws"]
+# [apis."aws"."kms".authentication]
+# [apis."aws"."kms".key_configuration]
```

Comment:
If authentication is empty, IDMS/IAM just works TM?

Reply:
Good question - I'll test and let you know

Reply:
Yeah it'll default to IAM if no keys are provided